native/Avalonia.Native/src/OSX/app.mm

```diff
@@ -24,9 +24,38 @@ - (void)applicationDidFinishLaunching:(NSNotification *)notification
 
 @end
 
+@interface AvnApplication : NSApplication
+
+
+@end
+
+@implementation AvnApplication
+{
+    BOOL _isHandlingSendEvent;
+}
+
+- (void)sendEvent:(NSEvent *)event
```

Comment:
looking at this code https://bitbucket.org/chromiumembedded/java-cef/src/e9a0b2c93cd913cd800aac940d1e709f1602fb07/native/util_mac.mm

you're also missing:
setHandlingSendEvent

Reply:
It's not called by Chromium or CEF code. It's only required by helper `CefScopedSendingEvent` C++ class that also isn't used by CEF and is there for convenience purposes.

Reply:
Got an exception because of that missing bit

Reply:
Crashing on exception: -[AvnApplication setHandlingSendEvent:]: unrecognized selector sent to instance 0x7fb2a3595f90